### DISCUSSION TITLE
Where's the data showing the unexplained deaths in healthy individuals post-jab?

### DISCUSSION THREAD

**Discussion by bronxnz** [owid/covid-19-data#2823] [General]
Where's the data showing the unexplained deaths in healthy individuals post-jab?

**Comment by lucasrodes** [comment]
Hi @bronxnz and thanks for the interest in our project.

We do not have indicators on that matter at the moment.

You can fins all our data on COVID-19 at https://ourworldindata.org/coronavirus
